blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientConfig.scala

```diff
@@ -37,6 +38,7 @@ case class BlazeClientConfig( //
                               maxResponseLineSize: Int,
                               maxHeaderLength: Int,
                               maxChunkSize: Int,
+                              isLenient: Boolean,
```

Comment:
I've recently been bitten by some booleans that turned into 3VL.  Should we get into the habit of calling these things `Strict` and `Lenient` instead of `true` and `false`?